client/src/components/Form/RequestForm.js

```diff
@@ -5,36 +5,43 @@ import {
     TextareaAutosize as Textarea,
 } from '@material-ui/core'
 import useMediaQuery from '@material-ui/core/useMediaQuery'
-import { useTheme } from '@material-ui/core/styles'
+import { useTheme, makeStyles } from '@material-ui/core/styles'
 import { connect } from 'react-redux'
 import {
     Actions,
     Select, MenuItem,
     Content, ContentText, Text,
 } from './components'
-import Tags from '../Tags/Tags'
+import { SelectedChip, UnselectedChip, DottedChip } from '../Tags/components'
+import { defaultTags } from '../../constant'
 import classes from '../../modules/form.module.css'
-import { addRequest, closeForm } from '../../actions'
+import { addRequest, closeForm, addBusiness } from '../../actions'
 
 const Form = ({
     open = false, closeForm, addRequest, owns = [],
 }) => {
     const [business, setBusiness] = useState('')
     const [details, setDetails] = useState('')
-    const [tags, setTags] = useState([])
+    const [tags, setTags] = useState(defaultTags)
     const fullScreen = useMediaQuery(useTheme().breakpoints.down('sm'))
 
     const onClose = () => {
         setDetails('')
-        setTags([])
+        setTags(defaultTags)
         closeForm('request')
     }
 
     const onSubmit = e => {
-        addRequest({ business, details, tags })
+        addRequest({ business, details, tags: tags.filter(({ selected }) => selected).map(({ label }) => ({ label })) })
+        addBusiness({ tags: tags.filter(({ selected }) => selected).map(({ label }) => ({ label })) })
```

Comment:
what does this do? should call `updateBusiness` after retrieving `_id` of the business this request should belong to

Reply:
Took off that line. Seems like updateBusiness needs to be implemented on actions as well. I'll update it when Edit business task is done :)

Reply:
that works! I'd just make sure to note it down somewhere